rest/route/github.go

```diff
@@ -280,6 +281,20 @@ func (gh *githubHookApi) Run(ctx context.Context) gimlet.Responder {
 			}))
 			return gimlet.MakeJSONErrorResponder(errors.Wrap(err, "can't enqueue item on commit queue"))
 		}
+
+		pushJob := units.NewGithubStatusUpdateJobForPushToCommitQueue(userRepo.Owner, userRepo.Repo, baseBranch, PRNum)
+		pushJob.Run(ctx)
```

Comment:
I think it would be better to add this to the local queue rather than blocking the web request on it. That way an unavailable GitHub status API doesn't stop us from returning 200 to the request.

Reply:
Done.